#1580 Improvement describe information about scada on top

Patrykb0802 opened the pull request:
Added:
- Custom text on top inside mainHeader, next to app logo
- input area for text to be displayed in main header next to logo
- input area for css style of that aforementioned custom text

Patrykb0802 committed:
#1580 Improvement describe information about scada on top

Added:
- Custom text on top inside mainHeader, next to app logo
- input area for text to be displayed in main header next to logo
- input area for css style of that aforementioned custom text

Patrykb0802 committed:
#1580 Improvement describe information about scada on top

Patrykb0802 committed:
#1580 Improvement describe information about scada on top

- Added custom information in system settings for new UI
- Added prefix for custom information
- Added css styles for custiom information se they are displayed properly
- Moved stylesheet to common custom stylesheet instead of special one onyl for custom information
- Added migration class V2_8_0_2_AddCustomInformationColumnsToSystemSettings.java

Limraj reviewed (commented):
This name is too long(remove 'to display'), and first prefix field, alternatively, consider changing the name to something more readable, e.g. 'Top description prefix' and 'Top description'.:
![image](https://github.com/user-attachments/assets/c2c2f36e-8c9d-40ec-960a-a2bcd54d4225)


It should look similar to our test server, it doesn't look very good at the moment. Give the html structure and styles from test server. 
![image](https://github.com/user-attachments/assets/f75d1991-b917-4c4b-afe7-e2dce1c6c72b)

Patrykb0802 committed:
#1580 Improvement describe information about scada on top

Improvements in css files

Patrykb0802 committed:
#1580 Improvement describe information about scada on top

Removed:
 - creating a column for stylesheet in migration class (it should be implemented in different issue)

Patrykb0802 committed:
#1580 Improvement describe information about scada on top

- Minor changes to css stylesheet

Patrykb0802 committed:
#1580 Improvement describe information about scada on top

- Changed way of top description being dynamically refreshed

Patrykb0802 committed:
#1580 Improvement describe information about scada on top

Limraj committed:
#1580 Improvement describe information about scada on top:
- corrected translation
